docs/guide.md

```diff
@@ -439,3 +439,13 @@ if (!storedToken) {
   localStorage.setItem('infos', JSON.stringify(infos))
 }
 ```
+
+### Reset a client
+
+When a user logout, we would like remove all reference on this instance and
+remove all user data. You can just use `cozyClient.unregister()`.
```

Comment:
If it is meant to reset the `client`, could we call the function `reset()`. I find `unregister` to be a bit unnatural.